duckduckgo/iOS
# Fix build settings resolution for external developers

**Description**:

[The README recommends](https://github.com/duckduckgo/iOS#developer-details) using an `ExternalDeveloper.xcconfig` file in order to supply values that differ from the `DuckDuckGoDeveloper.xcconfig`. The problem is that the project's targets have build setting customizations that override the `ExternalDeveloper.xcconfig` values. This PR removes target build settings that override developer configuration files. Allowing external developers to run the app without needing to modifying the project file.

**Steps to test this PR**:
1. Follow the steps in the ["Developer details" section of the README](https://github.com/duckduckgo/iOS#developer-details)
2. Open Xcode and verify the build settings for the target reflect the values assigned in `ExternalDeveloper.xcconfig` (particularly `DEVELOPMENT_TEAM`)

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
